eng/pipelines/apiview-review-gen-cadl.yml

```diff
@@ -44,9 +44,9 @@ jobs:
     condition: eq(variables['APIViewURL'], '')
 
   - pwsh: |
-      npm install -g cadl
-      npm install -g @cadl-lang/compiler
-      npm install -g @azure-tools/cadl-apiview
+      npm install -g @cadl-lang/compiler@0.37.0

```

Comment:
Pinned version due to a known issue in latest cadl-compiler

Reply:
Do you not need to install the cadl package any longer? Or is that a transitive dependency of cadl-lang/compiler? Similar if it is a dependency should we be able to only install the cadl-apivew package as I assume it already has a dependency on the compiler.

Reply:
Nope. It was a mistake. cadl package is not required. That's correct. Currently @cadl-lang/compiler is not set as a dependency of cadl-apiview so we need to explicitly install it. It's only a dev dependency. I will ask Travis to check if it can be regular dependency.